router/src/validation.rs

```diff
@@ -624,6 +767,26 @@ fn image_tokens(
     }
 }
 
+fn video_tokens(config: &Config, height: u32, width: u32, sampled_frames: f32) -> String {
+    use Config::*;
+
+    match config {
+        Qwen2Vl(_config) => {
+            let min_frames = 2_f32;
+            let max_frames = 256_f32;
+            // make sure the frames are within the range and are even
+            let nframes = (sampled_frames).max(min_frames).min(max_frames);
+            let nframes = (nframes / 2.0).round() as usize * 2;
+            let num_tokens = nframes * height as usize * width as usize / 1541;
```

Comment:
Keep everything in usize and use regular division or  `div_ceil`. Should be much simpler.

Where is that `1514` coming from ?